libs/media-player/src/lib/shared/use-key-down.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useEffect } from 'react';
+import { KeyboardShortcuts } from '../audio-annotator';
+
+export enum KeyboardKey {
```

Comment:
We can add more potential shortcuts as needed when we get to having configurable shortcut keys.